apps/src/aiComponentLibrary/chatItems/types.ts

```diff
@@ -0,0 +1,5 @@
+/** Identifies the role of a sender for a given chat message. */
```

Comment:
oh nice! I like the idea of sharing types where we can too

Reply:
Yep! Figured this one at least made sense to share, but we can also find a better place to consolidate these if/when we share more types